package com.my.study.oop.exception;

public class ThrowDemo {
    public static void main(String[] args) {
        throw new RuntimeException("运行时异常");
    }
}
